spec/models/user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe User, type: :model do
+  it { is_expected.to have_many(:photos).dependent(:destroy)  }
+  it { is_expected.to have_many :liked_photos }
+
+
+  describe '#owns?' do
+    it 'returns true if the photo is owned by the user' do
+      user = User.create(username: 'test', email: 'email@email.com', password: '1234pass')
+      photo = Photo.new(image_file_name: nil)
+      photo.user_id = user.id
+      photo.save
```

Comment:
Great that you've unit tested this logic. I'd see if you could possibly tidy this up. Also the fact you've got logic like this in your unit test should be a code smell - you can either use factory girl here or if there was a a method `Photo.create_with_user` you wouldn't have to do all this logic in the spec